src/base/system_parameter.h

```diff
@@ -476,6 +476,8 @@ enum param_id
   PRM_ID_REPLICATION_COMPRESS_PAGES_TRANSFER,
   PRM_ID_SCAL_PERF_PTS_REPL_THREAD_IGNORE_UNFIX,
   PRM_ID_SCAL_PERF_PS_REPL_THREAD_IGNORE_UNFIX,
+  PRM_ID_SCAL_PERF_PS_REQ_RESPONDER_THREAD_COUNT,
+  PRM_ID_SCAL_PERF_PS_REQ_RESPONDER_TASK_COUNT,
```

Comment:
The identifiers of these parameter might need to be changed later.
They have been added as a trial run during performance investigations. That's why they are cryptically called with `_SCAL_PERF_`.